File: LAB2/OP/5/csharp lab5/csharp lab5/TCylinder.cs
using System;
namespace csharp_lab5
{
    internal class TCylinder : TFigure
    {
        private double r, h;
        public TCylinder(double r, double h)
        {
            this.r = r;
            this.h = h;
        }
        
        public override double V()
        {
            return Math.PI * h * Math.Pow(r, 2);
        }
        public override double S()
        {
            return 2 * Math.PI * r * (h + r);
        }
        
        public double R { get => r; set => r = value;}
        public double H{ get => h; set => h = value;}
        
        public override string ToString()
        {
            return "Cylinder: " +
                   $"radius = {r} " +
                   $"height = {h} " +
                   $"Volume = {Math.Round(V(),3)}";
        }
    }
}

File: LAB2/OP/5/py/TCylinder.py
import math

from TFigure import *


class TCylinder(TFigure):

    def V(self):
        return math.pi * self.h * (self.r ** 2)

    def S(self):
        return 2 * math.pi * self.r * (self.h + self.r)

    def __init__(self, h, r):
        self.h = h
        self.r = r

    def __str__(self):
        return "Cylinder: " + f"radius ={self.r} " + f"height = {self.h} " + f"Volume = {round(self.V(), 3)} "
